supabase/migrations/20230809162431_schema.sql

```diff
@@ -41,6 +70,14 @@ end;
 $function$
 ;
 
+create policy "Enable insert for authenticated users only"
+on "public"."prefix_urls"
+as permissive
+for insert
+to authenticated
+with check (true);
```

Comment:
Not sure if we need to do anything to ensure created_by matches the current user?